app/serializers/errors_serializer.rb

```diff
@@ -4,8 +4,12 @@ def initialize(model)
   end
 
   def errors_hash
-    error_messages = @model.errors.to_hash(true).map do |_, message|
-      message.join
+    if @model.errors.messages.empty?
+      error_messages = ["credentials are bad"]
```

Comment:
Hardcoded in 🤷‍♀️ Might return to this later to see if other sad paths give me different responses